identity/app/views/consentJourney.scala.html

```diff
@@ -175,15 +175,35 @@ <h3 class="manage-account__switch-title">&nbsp;</h3>
 
 @confirmModal = {
     <section class="identity-forms-message">
-
+        @if(onlySubscribedToV1Newsletters.isEmpty) {
+            <h1 class="identity-header">Really?</h1>
+        }else {
+            <h1 class="identity-header">Are you sure?</h1>
+        }
         <div class="identity-forms-message__body">
-            You have not selected any newsletters or Guardian products and services.<br/>
-            This means you will stop receiving these emails from us. Are you sure you want to continue?
+            @if(onlySubscribedToV1Newsletters.isEmpty) {
```

Comment:
I would rather use one `@if` block here and duplicate the wrapper div that is common to both.